client/src/routes/game/+page.svelte

```diff
@@ -156,6 +163,25 @@
       }
     }
   }
+
+  $: privatePlayer, getCatfishes();
+  function getCatfishes() {
```

Comment:
shouldn't we call unsubscribeCatfishes here?

Reply:
I think I understand but explain why?

Reply:
I'm pretty sure the way the code is currently written we resubscribe every time the private player changes (each time there is a new prompt for example) but we never unsubscribe so that will put a lot of unwanted stress on the database.

Reply:
why is `unsubscribeCatfishes` not called if the state is ROLE?

Reply:
So you saying subscribe for that moment then unsubscribe? And a side note I think the getDocs function might be better since we're not using the newly updated documents to display to the user or anything

Reply:
No I'm saying unsubscribe before you subscribe

Reply:
Yeah getDocs makes sense to me

Reply:
> No I'm saying unsubscribe before you subscribe
Yeah, that makes sense.  You want to make sure you unsubscribe from the previous subscription before you subscribe again in the Role phases case.